src/pretty-dom.js

```diff
@@ -20,18 +20,34 @@ const getMaxLength = dom =>
 
 const {DOMElement, DOMCollection} = prettyFormat.plugins
 
-function prettyDOM(
-  dom = getDocument().body,
-  maxLength = getMaxLength(dom),
-  options,
-) {
```

Comment:
`inCypress` has failed with `null` parameter so in addition I replaced arguments default initializations.